cfg/config.yaml

```diff
@@ -155,6 +155,18 @@ llsfrb:
     refbox-host: localhost
     refbox-port: 4444
 
+  workpiece-tracking:
+   enable: true
+   fail-safe: true
+
+  #referee confirmation is usually required for some production steps
+  # delivery: auto/manual, A requested delivery in the real world needs to be
+  #                        confirmed 'manually' by the referee before scoring.
+  #                        In simulation this might be tedious and unnecessary.
+  #                        'Automatic' confirmation grants all delivery requests.
+  referee-confirmation:
+    delivery: manual
```

Comment:
Please make this a boolean, e.g., `auto-confirm-delivery: true` (no need for nesting)

(Also, I would have preferred to have this in a separate PR, this has nothing to do with this PR)

Reply:
I removed it from this PR, I will file another PR with your suggestion once #30 is merged into master.

The reason I had a nesting though, is that I think later might be useful to also specify how do we confirm a cap mount. Whether it should always be granted, confirmed by referee or confirmed on delivery. At the moment, we do not award the cap operation in its time due to the fact we can not tell if it was the correct cap color. Atm, the delivery confirmation is also used to determine the cap color correctness